archive.py

```diff
@@ -91,9 +94,12 @@ def get_config(section: str, key: str, default_value: Optional[Any]=None) -> Opt
 
 # writes the Jekyll header info for the index page listing all streams.
 def write_stream_index_header(outfile):
-    outfile.writelines(['---\n', 'layout: archive\n', 'title: Lean Prover Zulip Chat Archive\n'])
-    outfile.write('permalink: {}/index.html\n'.format(html_root))
-    outfile.writelines(['---\n\n','---\n\n','## Streams:\n\n'])
+    if is_lean_archive:
+        outfile.writelines(['---\n', 'layout: archive\n', 'title: Lean Prover Zulip Chat Archive\n'])
+        outfile.write('permalink: {}/index.html\n'.format(html_root))
+        outfile.writelines(['---\n\n', '---\n\n', '## Streams:\n\n'])
```

Comment:
I think we want this code to be unconditional, and just to get the title from the config file.

Reply:
Is it possible to parse title with newlines from the cfg file?

Reply:
To be clear, only the thing after `title:` should be conditional.  The rest is just reasonable headings for the page for everyone.

Reply:
Ok, I will retain the newlines for everyone.